content/en/docs/concepts/storage/volumes.md

```diff
@@ -1034,6 +1034,11 @@ As a result, operators do not have to make any
 configuration changes to existing Storage Classes, PersistentVolumes or PersistentVolumeClaims
 (referring to in-tree plugins) when transitioning to a CSI driver that supersedes an in-tree plugin.
 
+{{< note >}}
+* Existing PVs created by in-tree volume plugin can still be used in the future without any configuration changes even after the `CSIMigration` is completed and corresponing in-tree volume plugins are deleted.
+* New PVs with manifests that referring to in-tree volume plugins can still be created, even after the in-tree volume plugins are removed from Kubernetes.
```

Comment:
nit: `that referrig to` -> `referring to` or `that refer to`
Perhaps my last comment suggested it :-(

another nit:: a manifest includes PV(s) so, I personally prefer to "PVs in manifests" over "with manifests".

Reply:
Thank you for you kind suggestion! I've corrected the second sentence as you pointed out.